File: engine/engine-model/src/main/scala/net/server/channel/packet/alliance/ExpelGuildPacket.scala
package net.server.channel.packet.alliance

import net.server.MaplePacket

class ExpelGuildPacket( private var _guildId: Int,  private var _allianceId: Int) extends AllianceOperationPacket {
     def guildId: Int = _guildId
     def allianceId: Int = _allianceId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/alliance/ExpelGuildPacket.java
package net.server.channel.packet.alliance;

public class ExpelGuildPacket extends AllianceOperationPacket {
   private final int guildId;

   private final int allianceId;

   public ExpelGuildPacket(int guildId, int allianceId) {
      this.guildId = guildId;
      this.allianceId = allianceId;
   }

   public int guildId() {
      return guildId;
   }

   public int allianceId() {
      return allianceId;
   }
}
